projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -47,6 +47,12 @@ export const backends = [
     preview: boolean
 }[]
 
+export const editions = [
```

Comment:
I think an object should be used rather than an array here to allow us some abstraction between out understanding of the endpoint and the actual slug. e.g. `{weekly: 'australian-edition'}`